index.js

```diff
@@ -79,7 +79,7 @@ const fallback = {
 	circleCross: '(×)',
 	circlePipe: '(│)',
 	circleQuestionMark: '(?)',
-	bullet: '*',
+	bullet: '●',
```

Comment:
Can you do like https://github.com/sindresorhus/figures/blob/fba7e787b40948557ec63f0d8a1666739eef8927/index.js#L22 ? It makes it easier to read the code as you can easily see that they're the same.

Reply:
Did you mean using:

```js
	bullet: main.bullet,
```

If so, I can do this :+1: 

I can also update it in the other PRs.

Also, I can submit an additional PR to use this pattern for all characters that are common in `main` and `windows`. At the moment, there are still a few which are identical but do not use this pattern (for example `line`).

Reply:
Yeah, that's what I meant. I must have copied the wrong URL.

Reply:
> Also, I can submit an additional PR to use this pattern for all existing characters that are common in main and windows

Sure. I think one PR for all those changes is best.

Reply:
Fixed!

It turns out `line` was the only figure not using this pattern. I submitted a PR at https://github.com/sindresorhus/figures/pull/44
I also fixed all the current PRs not using this pattern :+1: